### PR TITLE
Draft of release notes and related changes for the efm 4.6 release.

### PR DESCRIPTION
**Pull request by EFM-Bobby**
Some updates for efm 4.6 release.

### CODE REVIEW

**Comment by EFM-Bobby** [comment]
@drothery-edb This isn't ready to go live yet (and shouldn't before 14 Feb), but can you tell me if I've created the link correctly for the new 4.6 release announcement page? I think I remember some discussion about a different way to handle this that was going to happen, but I don't recall if it did.

File: product_docs/docs/efm/4/10_notifications.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Notifications"
-redirects: 
+redirects:
  - ../efm_user/10_notifications
```
Comment (EFM-Bobby) [inline product_docs/docs/efm/4/10_notifications.mdx:4]: I don't know how to make my editor not do these troublesome whitespace "fixes," but this and a couple other places are just whitespace changes.

**Comment by drothery-edb** [comment]
@EFM-Bobby thanks for drafting the release notes. I made a couple tweaks. Your link from the release announcement looks good.

**Comment by EFM-Bobby** [comment]
@drothery-edb Am just curious -- are we adding "now supports epas 15" to all the release notes? Previous versions of efm also work with it, so this isn't new for this version of efm.

**Comment by EFM-Bobby** [comment]
@drothery-edb The efm packages are all live now. Can you merge this when you have a chance?

**Comment by martinmarques** [comment]
@kelpoole @drothery-edb would it be possible to get these release notes reviewed and pushed to the docs later today?

**Review by drothery-edb — APPROVED** [review]

**Comment by drothery-edb** [comment]
LGTM - I'll merge the PR and the changes will be published today.

**Pull request merged**